src/judges/codeforces/getContests.test.js

```diff
@@ -3,12 +3,12 @@ const request = require('request');
 import { getContests } from './getContests';
 
 function testContests(...contests) {
-  request.mockImplementation((url, callback) => {
-    setTimeout(() => callback(null, {statusCode: 200}, JSON.stringify({
+  request.mockReturnValueOnce(new Promise(function(resolve) {
+    setTimeout(() => resolve(JSON.stringify({
```

Comment:
why do we need a timeout?

Reply:
i think this is not related to your commit, but you are mocking the wrong library

Reply:
i will open a new issue regarding these tests

Reply:
I'll make it resolve right away.